inline-r/tests/tests.hs

```diff
@@ -105,4 +112,18 @@ tests = testGroup "Unit tests"
 main :: IO ()
 main = do
     _ <- R.initialize R.defaultConfig
-    defaultMain tests
+    argv <- getArgs
+    -- Assume gctorture() step size of 1.
+    let tortureCLI = "1" <$ find (== "--torture") argv
+    tortureEnv <- lookupEnv "R_GCTORTURE"
+    torture <- case tortureCLI <|> tortureEnv of
+      Nothing -> return False
+      Just x -> do
+        -- gctorture turned on. So assume moreover --enable-strict-barrier.
+        let step = read x :: Int32
+        putStrLn "WARNING: gctorture() turned on.\n\
+                 \    Tests will fail if R not compiled with --enable-strict-barrier."
```

Comment:
This seems to claim there will be false positives, when the patch really stops running tests to don't get false negatives.

Maybe:
```Haskell
putStrLn "WARNING: gctorture() turned on.\n\
                  \    Tests might pass when they shouldn't if R is not compiled with --enable-strict-barrier."
```